### ISSUE TITLE
ARIA Practice Listbox Example with Grouped Options does not navigate past groupings in NVDA browse mode on Chrome

### ISSUE THREAD

**Issue by sstorck** [nvaccess/nvda#20830]
### Brief summary

When browsing a listbox with groups, NVDA does not read out more than 1 group.

Also filed https://github.com/w3c/aria-practices/issues/3470 against the aria-practices project since it's not clear to me whether this is an NVDA bug or a bug in the example code.

### Steps to reproduce

1. Open Chrome.
2. Navigate to https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-grouped/
3. In browse mode, navigate to the listbox.
4. Down arrow through the options in the listbox.

### Actual behavior

Exits the listbox after the Land grouping. Land grouping is reported as clickable.

NVDA speech viewer output:

> Choose your animal sidekick
> list clickable Land grouping Land
> list clickable Cat
> list clickable Dog
> list clickable Tiger
> list clickable Reindeer
> list clickable Raccoon
> out of grouping separator
> heading level 2 Accessibility Features

### Expected behavior

Reads out every item in the listbox. Only animals are clickable.

### NVDA logs, crash dumps and other attachments

[nvda.log](https://github.com/user-attachments/files/32068260/nvda.log)

### NVDA type

installed copy

### NVDA version

2026.2 (2026.2.0.57664)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

No

### Windows version

Version	10.0.19045 Build 19045

### Name and version of other software in use when reproducing the issue

Chrome version: 152.0.7977.83 (Official Build) (64-bit) (cohort: Stable) 

### Does the issue still occur after restarting your computer?

I have not restarted my computer

### If NVDA add-ons are disabled, is your problem still occurring?

I have not restarted NVDA with add-ons disabled

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool
